internal/tablelist.go

```diff
@@ -37,16 +37,22 @@ type TableList interface {
 }
 
 type tableList struct {
-	pdf  MarotoGridPart
-	text Text
-	font Font
+	pdf        MarotoGridPart
+	text       Text
+	font       Font
+	maxGridSum float64
```

Comment:
I think that this should be an int.

Reply:
but the default as well as all related setter functions expect a float64 value?

Reply:
You are right. Could you just add the example with this custom grid sum?